phenotrex/ml/trex_classifier.py

```diff
@@ -9,26 +9,35 @@
 import numpy as np
 
 from sklearn.base import clone
-from sklearn.metrics import balanced_accuracy_score, accuracy_score, f1_score
+from sklearn.metrics import balanced_accuracy_score, f1_score, precision_score, recall_score
 from sklearn.model_selection import StratifiedKFold, LeaveOneGroupOut, RandomizedSearchCV
 from sklearn.feature_selection import RFECV
+from sklearn.feature_extraction.text import CountVectorizer
 
-from phenotrex.ml.vectorizer import CustomVectorizer
 from phenotrex.structure.records import TrainingRecord, GenotypeRecord
 from phenotrex.ml.cccv import CompleContaCV
 from phenotrex.util.helpers import get_x_y_tn_ft, get_groups
 from phenotrex.ml.feature_select import recursive_feature_elimination, DEFAULT_STEP_SIZE, \
     DEFAULT_SCORING_FUNCTION
 
 
+def specificity_score(y, y_pred, **kwargs) -> float:
+    """
+    Method for binary classification to get specificity: recall of the negative class
+    """
+    return recall_score(y, y_pred, pos_label=0, **kwargs)
+
+
 class TrexClassifier(ABC):
     """
     Abstract base class of Trex classifier.
     """
     scoring_function_mapping = {
-        'accuracy': accuracy_score,
         'balanced_accuracy': balanced_accuracy_score,
-        'f1': f1_score
+        'f1': f1_score,
+        'precision': precision_score,
+        'recall': recall_score,
+        'specificity': specificity_score
```

Comment:
It's always nice to have a comma also in the last line. Good for future diffs.
```suggestion
        'specificity': specificity_score,
```